### PR TITLE
docs: first draft of ksqlDB events overview topic (DOCS-2927)

### PR DESCRIPTION
**Pull request by JimGalasyn**

### CODE REVIEW

File: docs-md/concepts/events.md [hunk 1]
```diff
@@ -1,15 +1,53 @@
...
+high-level that happened in a business, like the sale of an item or the
+submission of an invoice. Or it could be something low-level, like a log line
+emitted by a web server when a request is received. Anything that happens at
+a point in time is an event.
 
-Apache Kafka is a distributed streaming platform for working with events. It’s horizontally scalable, fault-tolerant, and wicked fast. Although working with it directly can be low level, it has an excellent opinionated approach for modeling both individual events and groups of them together. For this reason, ksqlDB heavily borrows some of Kafka’s abstractions. It doesn’t aim to make you learn all of Kafka, but it also doesn’t reinvent the wheel where there’s already something really good to use.
+Because events are so fundamental to stream processing, they are ksqlDB's core
+unit of data. All of ksqlDB's features are oriented around making it easy to
+solve problems using events. Although it's easy to think about individual
+events, figuring out how to group related events together is a bit more
```
Comment (vinothchandar) [inline docs-md/concepts/events.md]: relating back to the comment on the other PR .. I was also suggesting grouped -> stored, to make it clearer when we discuss aggregating/grouping here ultimately

File: docs-md/concepts/events.md [hunk 2]
```diff
@@ -1,15 +1,53 @@
...
+challenging. Fortunately, the idea of storing related events is well-explored
+territory. {{ site.aktm }} leads the way, which is why ksqlDB is built directly
+on top of it.
 
-ksqlDB represents events with a simple key/value model, which is very similar to Kafka’s notion of a record. The key represents some form of identity about the event. The value represents information about the event that occurred. This combination of key and value makes it easy to model groups of events, since multiple events with the same key represent the same identity, irrespective of their values. Events in ksqlDB carry more information than just a key and value, though. Similar to Kafka, they also describe the time at which the event was true.
+{{ site.ak }} is a distributed streaming platform for working with events. It’s
+horizontally scalable, fault-tolerant, and extremely fast. Although working
+with it directly can be low-level, it has a strong and opinionated approach for
```
Comment (vinothchandar) [inline docs-md/concepts/events.md]: can we find a better phrase than low-level?  (I dont have ideas either :))

Comment (vinothchandar) [reply docs-md/concepts/events.md]: We never discuss Kafka Streams here at all? Is that a good idea?

File: docs-md/concepts/events.md [hunk 3]
```diff
@@ -1,15 +1,53 @@
...
+title: Events
+tagline: Events in ksqlDB
+description: Learn about events and stream processing in ksqlDB. 
+keywords: ksqldb, event, stream
 ---
 
-ksqlDB is an event streaming database purpose-built for stream processing applications. The main focus of stream processing is modeling computation over unbounded streams of events. But what is an event anyway?
+ksqlDB is an event streaming database that's purpose-built for stream
+processing applications. The main focus of stream processing is modeling
+computation over unbounded streams of events.
```
[the remark this answers is not in the document]
Comment (MichaelDrogalis) [reply docs-md/concepts/events.md]: @JimGalasyn Can we move this back under the header? I agree with Vinoth that lifting this up makes the doc read a little funny. It's meant to connect to saying why we care about events.

File: docs-md/concepts/events.md (hunk 1 above)
Comment (MichaelDrogalis) [reply docs-md/concepts/events.md]: Cool. +1. lll use that language going forward.

File: docs-md/concepts/events.md (hunk 2 above)
Comment (MichaelDrogalis) [reply docs-md/concepts/events.md]: My take is that it's an ok term. Jay's described Kafka a couple of times like this recently. But if we can find another term that conveys something similar, that's good too.

Comment (MichaelDrogalis) [reply docs-md/concepts/events.md]: I was referring to Apache Kafka in its entirety. I think we need to enhance the chapter of the docs on KSQL and Kafka Stream's relationship to one another. What was the thing about KStreams that you wish was said here? Perhaps we can make the dedicated KStreams chapter back a better punch.

Comment (vinothchandar) [reply docs-md/concepts/events.md]: Fair point. I read AK as Kafka - the storage system and was wondering if we have discussed the role Kafka Streams the stream processing system plays here.. I am find with digging deeper separately

**Pull request merged**